<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blackokno</class>
 <widget class="QDialog" name="blackokno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#006633</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>520</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Powr√≥t</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stankonta">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>370</y>
     <width>261</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="hitButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
     background-color: #F9E04B;
 padding: 5px;
}    
QPushButton:pressed{
     background-color:#F9E04B;
 padding: 5px;
}   </string>
      </property>
      <property name="text">
       <string>HIT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="dealButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
     background-color: #F9E04B;
 padding: 5px;
}    
QPushButton:pressed{
     background-color:#F9E04B;
 padding: 5px;
}   </string>
      </property>
      <property name="text">
       <string>DEAL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="standButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
     background-color: #F9E04B;
 padding: 5px;
}    
QPushButton:pressed{
     background-color:#F9E04B;
 padding: 5px;
}   </string>
      </property>
      <property name="text">
       <string>STAND</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="dealerHandEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="playerHandEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dealerScoreLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="playerScoreLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="betLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>RESET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
